sympy/functions/elementary/trigonometric.py

```diff
@@ -2261,6 +2295,20 @@ def eval(cls, arg):
         if i_coeff is not None:
             return S.ImaginaryUnit * atanh(i_coeff)
 
+        if isinstance(arg, tan):
+            ang = arg.args[0]
+            if ang.is_comparable:
+                ang %= pi # restrict to [0,pi)
+                if ang > pi/2: # restrict to [-pi/2,pi/2]
+                    ang = pi - ang
+
+                return ang
+
+        if isinstance(arg, cot): # atan(x) + acot(x) = pi/2
```

Comment:
atan(x) + acot(x) = pi/2 is true for some definition of acot. It is not true for the definition used by SymPy, mpmath, or WolframAlpha. All of them believe that acot is a discontinuous odd function, for example acot(-1) is -pi/4 rather than 3pi/4. 

So this, and its analog below, needs to be revised.